apps/dsky_clock/metadata.json

```
@@ -0,0 +1,17 @@
+{ "id": "dsky_clock",
+    "name": "DSKY Clock",
+    "shortName":"dsky_clock",
+    "icon": "dsky_clock.img",
+    "screenshots": [{"url":"screenshot1.png"}],
+    "version": "0.01",
+    "description": "A clockface inspired by the Apollo DSKY interface.",
+    "tags": "clock",
+    "type": "clock",
+    "supports":["BANGLEJS2"],
+    "readme": "README.md",
+    "allow_emulator": true,
+    "storage": [
+      {"name":"app.js","url":"app.js"},
```

```suggestion
      {"name":"dsky_clock.app.js","url":"app.js"},
```

(`name` is the filename that will be set on the watch, and `url` is the filename in the BangleApps repo)